src/XMakeTasks/Exec.cs

```diff
@@ -58,13 +58,25 @@ public Exec()
         private readonly List<ITaskItem> _nonEmptyOutput = new List<ITaskItem>();
         private Encoding _standardErrorEncoding;
         private Encoding _standardOutputEncoding;
+        private string _command;
 
         #endregion
 
         #region Properties
 
         [Required]
-        public string Command { get; set; }
+        public string Command
+        {
+            get { return _command; }
+            set
+            {
+                _command = value;
+                if (Path.DirectorySeparatorChar == '/')
```

Comment:
Could you also please replace all the occurences of `if (Path.DirectorySeparatorChar == '/')` with IsUnixLike? :)